app/src/main/java/com/example/whattowatch/NewMovieListActivity.java

```diff
@@ -95,6 +95,10 @@ public void done(ParseException e) {
         rvMovieList.setLayoutManager(new LinearLayoutManager(this));
     }
 
+    /**
+     *  onActivityResult is the method that runs when an activity comes back after a previous activity has finished
+     *  We are overwriting this method to get the movie we selected in the FindMovieActivity and adding it to our recyclerView
+     */
```

Comment:
You don't need to explain onActivityResult, but rather what onActivityResult does in the context of this activity. In this case it adds a new movie and notifies (notifies who?)